contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOInputOperator.java

```diff
@@ -61,6 +61,7 @@
 @Evolving
 public class CassandraPOJOInputOperator extends AbstractCassandraInputOperator<Object> implements Operator.ActivationListener<OperatorContext>
 {
+  private String TOKEN_QUERY;
```

Comment:
Please follow Java naming conventions. This field not a constant so it should be in camel case